### ISSUE TITLE
IAE doesn't start training if there's no IAE_encoder.h5 file already present.

### ISSUE THREAD

**Issue by Kirin-kun** [deepfakes/faceswap#276]
The model doesn't start training if there's no IAE_encoder.h5 file in the directory. It crashes with "file not found".

I renamed an original training file as IAE_encoder.h5 and it started.

It should create a new file if none is found, like the other models.

**Comment by Clorr** [comment]
Hmm, IAE should have the same behavior than other models. Also encoder file has the same behavior as other files so I don't see a reason why it would fail. It will output a warning anyway but it shouldn't stop... I can't test right now, if someone else can give more feedback it would be nice...

**Comment by JayantPythonLover** [comment]
It creates new files if no existing model is found. It gives you a warning for the same but continues to start training as expected. I just tested it. Seems to be working as intended.

This could be something on your end. Please double check that the folder you are writing to has write permissions.

**Comment by Kirin-kun** [comment]
Hmm, that's odd. When I first started it, I had a message:

Unable to open file (unable to open file: name = 'H:\Fakes\modeld.iae\IAE_encoder.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
Loading Trainer from Model_IAE plugin...

Then it stalled and didn't do anything. 

But I just retested, this time with a completely empty directory and it gave the same message, but started anyway. I guess it can be closed.
